### DISCUSSION TITLE
I have set the option debug-file=true but the log file size is always zero.

### DISCUSSION THREAD

**Discussion by YuChengLin817** [mapsforge/mapsforge#1406] [Q&A] [answered]
Hi,
I have set the option debug-file=true and I also modified the logging.properties of JRE but the log file size is always zero.
Is there anything wrong?

My JRE logging.properties like following:
...
handlers = java.util.logging.FileHandler, java.util.logging.ConsoleHandler
...
.level = TRACE

Thanks for help!

**Comment by devemux86** [comment, marked as answer]
You probably mean map-writer's [debug-file](https://github.com/mapsforge/mapsforge/blob/master/docs/Getting-Started-Map-Writer.md#advanced-options-only-use-when-you-know-what-you-are-doing) advanced option.

Debug information is written to the map file and increases the map size.
You can check the `MapFileWriter` class about the written debug info.

**Comment by YuChengLin817** [comment]
Understood! So the java.util.logging.Logger of Maps-Writer will not be
logging into JRE log file?

Emux ***@***.***> 於 2023年4月11日 週二 下午4:32寫道：

**Reply by devemux86** [reply]
The internal logs of map-writer are written in the console.
You can redirect the console output to a file with:
```console
SomeCommand > SomeFile.txt
```
There are also more command line options if search online.

**Comment by YuChengLin817** [comment]
I see, I though the maps-writer will apply jre logging.properties
configuration to output logs.
Thanks for your help :)

Emux ***@***.***>於 2023年4月11日 週二，下午6:34寫道：
